.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,9 +2,8 @@
 name: Bug report
 about: Help us improve by reporting a bug
 title: '[BUG]'
-labels: ''
+labels: ['bug']
```

Comment:
i don't think we want it to be automatically flags with labels. 

I just want the label to be set by someone with permission (we trust), rather than being set by default (otherwise it would probably flood the page)

Reply:
IMHO, Adding 'bug' labels would makes easier to identify issues rather than adding it to the title.